.vscode/extensions.json

```diff
@@ -0,0 +1,6 @@
+{
+  "recommendations": [
+    "esbenp.prettier-vscode",
+    "streetsidesoftware.code-spell-checker"
+  ]
+}
```

Comment:
Developers prefer personal settings and extensions. Is it a good idea to force these extensions?

Reply:
- Yes we do want to enforce these to have consistent formatting on all contributions, develops can create separate profiles in VSCode if they don't want this to conflict with their personal setup. 

- This also helps us avoid setting up pre-commit hooks to achieve the same thing.